cg/meta/workflow/nextflow_common.py

```diff
@@ -35,6 +35,13 @@ def get_case_config_path(cls, case_id: str, root_dir: str) -> Path:
         """Generates a path where the Rnafusion sample sheet for the case_id should be located."""
         return Path((cls.get_case_path(case_id, root_dir)), f"{case_id}_samplesheet.csv")
 
+    @classmethod
+    def get_case_pid_path(cls, case_id: str, root_dir: str) -> Path:
```

Comment:
```suggestion
    def get_case_nextflow_pid_path(cls, case_id: str, root_dir: str) -> Path:
```
There can be many PIDs, OS, SLURM, NF etc